examples/Reference Building - Primary Schoool.xml

```diff
@@ -8991,11 +9009,17 @@
 					<Addition/>
 				</TypeOfMeasure>
 				<SystemCategoryAffected>Lighting</SystemCategoryAffected>
-				<PremisesAffected>
-					<PremiseAffected IDref="Building1">
-						<MeasureCoverage>0.8</MeasureCoverage>
```

Comment:
If this is needed, then we can add it to the measures world. At the moment, I don't think removing this will cause any issues.